packages/flame/src/Alert/Alert.tsx

```diff
@@ -90,11 +94,19 @@ const Alert: React.FunctionComponent<AlertProps & SpaceProps> = ({
         )}
         <Box flex="1">
           {title && (
-            <Text color="textHeading" fontWeight="bold" fontSize="text" mt={0} mr={0} mb={1} ml={0}>
+            <Text
+              color="textHeading"
+              fontWeight="bold"
+              fontSize={['text-l', 'text']}
+              mt={0}
+              mr={0}
+              mb={1}
```

Comment:
Design needs to go back to the drawing board on this one, they put a 3px space. It either needs to be `space.0` or `space.1`, and align with the visuals.